lesson07/assignment/index.html

```diff
@@ -0,0 +1,208 @@
+          message: "Places to see and visit in Oregon ",
+          title: "Favorite things to do",
+          waLocations: [
+            { text: 'Canadian Border' },
+            { text: 'Washington Coast' },
+            { text: 'Mount Baker' },
+            { text: 'Mount Rainier' },
+            { text: 'Pow-wow Time' }
+          ],
+          orLocations: [
+            { text: 'Portland' },
+            { text: 'Oregon Coast' },
+            { text: 'Mount Hood' },
+            { text: 'Table Rock' },
+            { text: 'Pow-wow Time' }
+          ],
+          imagesWa: [{
+            id: 1,
+            src: "./images/CanoeJourney.jpg",
+            alt: "Canoe Journey Alki Beach"
+          },
+          {
+            id: 2,
+            src: "./images/haystack-rock-NeahBay.jpg",
+            alt: "Haystack Rock in the Pacific Ocean, Neah Bay, WA"
+          },
+          {
+            id: 3,
+            src: "./images/mt-baker-Washington.jpg",
+            alt: "Mount Baker east of Bellingham, Washington"
+          },
+          {
+            id: 4,
+            src: "./images/MtRainier.jpg",
+            alt: "Mount Raineir east of Seattle, Washington"
+          },
+          {
+            id: 5,
+            src: "./images/westport-crab-pots.jpg",
+            alt: "Westport on the Washington Coast"
+          },
+          {
+            id: 6,
+            src: "./images/Dawn_dancing_2.jpg",
+            alt: "University of Washington Spring Pow-wow"
+          },
+          {
+            id: 7,
+            src: "./images/BlueMountains-WallaWalla.jpg",
+            alt: "Blue Mountain Range in Walla Walla, WA"
+          }
+          ],
+          imagesOr: [{
+            id: 1,
+            src: "./images/PortlandOregon.jpg",
+            alt: "Portland, Oregon view of the city"
+          },
+          {
+            id: 2,
+            src: "./images/OregonCoast.jpg",
+            alt: "Oregon Coast in Lincoln City"
+          },
+          {
+            id: 3,
+            src: "./images/DePoeBay.jpg",
+            alt: "DePoe Bay, Oregon"
+          },
+          {
+            id: 4,
+            src: "./images/Pow-wow-time-siletz.jpg",
+            alt: "Confederated Tribes of Siletz Oregon Indigenous Pow-wow"
+          },
+          {
+            id: 5,
+            src: "./images/Pow-wow-time-siletz.jpg",
+            alt: "Siletz Oregon Annual Pow-wow"
+          },
+          {
+            id: 6,
+            src: "./images/PowWow_vets.jpg",
+            alt: "Grand Entry Time Annual Pow-wow"
+          },
+          {
+            id: 7,
+            src: "./images/Oregon-lighthouse.jpg",
+            alt: "Lighthouse on the Oregon Coast"
+          }
+          ],
+        }
+      },
+      methods: {
+        generateGreeting() {
+          return "Hello Welcome to the Destination Site!"
+        },
+        addOneWa() {
+          this.imageIndex = (this.imageIndex + 1) % this.imagesWa.length;
```

Comment:
The reason both images are changing is because you are re-using the same `this.imageIndex`. So regardless of whether you click the WA or OR buttons, you are adding 1 to your imageIndex therefore changing both. Try having a `waImageIndex` and `orImageIndex`